### PR TITLE
Corrected warning/issues as introduced by React 15.

### PR DESCRIPTION
**Pull request by vipoo**
* https://gist.github.com/jimfb/d99e0678e9da715ccf6454961ef04d1b

  Warnings were raised by React due to incorrect props being
  supplied to the underlying html elements
  (eg: attempting to create element 'div' with a prop 'center')

  Props used by the Grid, Col, and Row elements are stripped
  before being passed to the underlying elements.

* devDependency for React has been upgraded to 15.
* React warnings will now fail associated tests

Signed-off-by: Dean Netherton <dean.netherton@gmail.com>

### CODE REVIEW

File: src/components/Col.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+  smOffset: PropTypes.number,
+  mdOffset: PropTypes.number,
+  lgOffset: PropTypes.number,
+  reverse: PropTypes.bool,
+  className: PropTypes.string,
+  tagName: PropTypes.string,
+  children: PropTypes.node
+};
+
+const propKeys = Object.keys(propTypes);
```
Comment (heldr) [inline src/components/Col.js]: Nice commit! 😄 

Why don't move this Object.keys call to clearProps?

**Comment by heldr** [comment]
NVM, I will do it!

**Comment by hashwin** [comment]
Hi, any chance this can be pushed to npm? The warnings are clouding other errors during dev.

**Comment by heldr** [comment]
@roylee0704 could please publish on npm? I believe its a minor release

**Comment by heldr** [comment]
@roylee0704 Actually, could please review this https://github.com/roylee0704/react-flexbox-grid/pull/43 and then publish?

**Comment by heldr** [comment]
@roylee0704 Actually, could please review this #43 and then publish?

**Comment by NeXTs** [comment]
How it's going? Any problems with publishing?

**Comment by israelidanny** [comment]
This is a major issue for a project I'm working on, is there a schedule known for the release of this patch?

**Comment by dnetherton** [comment]
For those interested, you can specify a github reference to a commit - in my package.json file, I use 

```"react-flexbox-grid": "vipoo/react-flexbox-grid#deans-dev"```

**Comment by roylee0704** [comment]
@heldr my machine was sent to repair. How can I help?

**Comment by NeXTs** [comment]
@roylee0704 
`npm publish`, please

**Comment by roylee0704** [comment]
@NeXTs okay, sorry for the wait. Let me find a machine  to do that.

**Comment by NeXTs** [comment]
@roylee0704 Sure, no hurry

**Comment by roylee0704** [comment]
@NeXTs  Okay the new version is up! =) 

cc: @heldr, @silvenon, @vipoo, @hashwin, @israelidanny.

**Comment by NeXTs** [comment]
newly released package doesn't contains lib folder

nolib

**Pull request merged**